modules/installing-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/install-aws-load-balancer-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-installing-aws-load-balancer-operator_{context}"]
+= Installing the AWS Load Balancer Operator
+
+You can install the AWS Load Balancer Operator using the Red Hat {product-title} Operator Hub.
+
+.Prerequisites
+
+* You have logged in to the {product-title} web console as a user with `cluster-admin` permissions.
```

Comment:
Additional prerequisites are needed. I've added the needed docs [here](https://github.com/openshift/aws-load-balancer-operator/pull/61), You pull them in once that PR is merged.

Reply:
There are already instructions in the CSV which include instructions to create the CredentialsRequests. I would like to see that all of them are in sync. Right now we are heading to a situation where we have multiple sets of instructions all linked to non-versioned files in the repository. I propose that we have a single file which has instructions. We don't include instructions on how the `CredentialsRequest` should be processed in an STS cluster, just link to the original documentation.

Reply:
@xenolinux For now, you can ignore the PR and just reference that the additional prerequisites are mentioned in the CSV during install.

Reply:
okay